libs/dh/shared/domain/src/lib/generated/v1/model/user-invite-audit-log-dto.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * DataHub BFF
+ * Backend-for-frontend for DataHub
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+export interface UserInviteAuditLogDto { 
+    userId: string;
+    changedByUserId: string;
+    actorId: string;
+    actorName: string;
+    timestamp: string;
+}
```

Comment:
question: this DTO doesn't look like it's being used. Can it be removed (no manually but by deleting `UserInviteAuditLogDto.cs`)?